6_visualize/src/create_gif_tasks.R

```diff
@@ -15,20 +15,27 @@ create_intro_gif_tasks <- function(n_timesteps, folders){
 
   tasks <- tidyr::crossing(timestep, cfgs) %>%
     unite(task_name, cfgs, timestep, sep = '_', remove = F) %>%
-    mutate(date_hour = sprintf('00_%03d', timestep),
+    mutate(date_hour = sprintf('01_%03d', timestep),
            task_name = sprintf("%s_%s", cfgs, date_hour))
 
   gage2spark <- scipiper::create_task_step(
     step_name = 'gage2spark',
     target_name = function(task_name, step_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
-      sprintf('gage2spark_%s', task_name)
+      sprintf('gage2spark_fun_%s', task_name)
     },
     command = function(task_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
       psprintf(
         "prep_gage2spark_fun(",
-        "timestep=I(%d))" = cur_task$timestep
+        "intro_config = intro_config,",
+        "timestep=I(%d)," = cur_task$timestep,
+        "storm_data = storm_data,",
+        "site_data = site_data,",
```

Comment:
True that the row contents are redundant, but I want to keep `site_data` here because I want to know and have the option of preserving the exact order of the sites as ordered in site_data. It's possible that future changes to storm_data might rearrange the sites, so it seems a bit more traceable to me to use site_data directly rather than relying on storm_data to also preserve that order.

In practice, I'm currently rearranging sites within `prep_spark_funs_data` anyway, but I think I can imagine scenarios where we'd want to do more in the site_data function to customize that ordering - in particular, the current ordering in site_data more or less groups by major river, which produces a pretty neat effect in the intro animation that we might want to experiment more with.